docs/source/Installation/compilation.md

```diff
@@ -14,6 +14,10 @@ For Mac OS X we have used the *gfortran* compiler installed together with GCC us
 
 Other package managers like [MacPorts](https://www.macports.org) or [Fink](http://www.finkproject.org) might also work, but we have not tried. 
 
+There is currently a known issue involving the compilation of the PDD solver on new Macs with M1/ARM/Apple silicon chips. 
```

Comment:
This is a total pain for developers. Nothing works out of the box with the new chip...

Reply:
It really is annoying. S4 was surprisingly easy to compile, I was very worried about that... But it is very annoying that to install both S4 and the PDD, I have to install gcc via homebrew, compile S4, uninstall it, install another gfortran, compile the PDD, and then re-install gcc via homebrew. Not user-friendly at all.